2022-01-14T17:55:18.434+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"user-web.json","group":"pro","tenant":"c1872978-d51c-4188-a497-4e0cd20b97d5"}>, call domain error:<Get "http://192.168.0.104:8848/nacos/v1/cs/configs?dataId=user-web.json&group=pro&tenant=c1872978-d51c-4188-a497-4e0cd20b97d5": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2022-01-14T17:55:18.435+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"user-web.json","group":"pro","tenant":"c1872978-d51c-4188-a497-4e0cd20b97d5"}>, call domain error:<Get "http://192.168.0.104:8848/nacos/v1/cs/configs?dataId=user-web.json&group=pro&tenant=c1872978-d51c-4188-a497-4e0cd20b97d5": dial tcp 192.168.0.104:8848: connect: no route to host> , result:<>
2022-01-14T17:55:18.435+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"user-web.json","group":"pro","tenant":"c1872978-d51c-4188-a497-4e0cd20b97d5"}>, call domain error:<Get "http://192.168.0.104:8848/nacos/v1/cs/configs?dataId=user-web.json&group=pro&tenant=c1872978-d51c-4188-a497-4e0cd20b97d5": dial tcp 192.168.0.104:8848: connect: host is down> , result:<>
2022-01-14T17:55:18.435+0800	[34mINFO[0m	get config from server error:Get "http://192.168.0.104:8848/nacos/v1/cs/configs?dataId=user-web.json&group=pro&tenant=c1872978-d51c-4188-a497-4e0cd20b97d5": dial tcp 192.168.0.104:8848: connect: host is down 
2022-01-14T17:55:18.435+0800	[31mERROR[0m	get config from cache  error:failed to read config cache file:tmp/nacos/cache/config/user-web.json@@pro@@c1872978-d51c-4188-a497-4e0cd20b97d5,err:open tmp/nacos/cache/config/user-web.json@@pro@@c1872978-d51c-4188-a497-4e0cd20b97d5: no such file or directory  
